services/document_conversion/v1.js

```diff
@@ -0,0 +1,120 @@
+module.exports = function(RED) {
+
+	var http = require('http');
+	var https = require('https');
+	
+	var cfEnv = require("cfenv");
+	var util = require("util");
+		
+	var watson = require('watson-developer-cloud');
+	var temp = require('temp');
+	temp.track();
+	var fs = require('fs');
+	var fileType = require('file-type');
+	var request = require('request');
+
+	var appEnv	 = cfEnv.getAppEnv();
+
+	var converts = [];
+		
+	// GNF: Read credentials information from the VCAP environment variable
+	for (var i in appEnv.services) {
+		// filter the services to include only the Convert ones
+		if (i.match(/^(document_conversion)/i)) {
+			converts = converts.concat(appEnv.services[i].map(function(v) {
+				return {
+					name: v.name, 
+					label: v.label, 
+					url: v.credentials.url,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2386925629/issues/source?bid=3225115&fileBranchId=3225117#l28)